lifelines/fitters/aalen_additive_fitter.py

```diff
@@ -415,6 +415,38 @@ def predict_expectation(self, X):
         t = self.cumulative_hazards_.index
         return pd.DataFrame(trapz(self.predict_survival_function(X)[index].values.T, t), index=index)
 
+    @property
+    def summary(self):
```

Comment:
usually summary has results like standard errors and p-values, I like that consistency. I would prefer to have p-values in the model before defining a summary functtion.